feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio.c

```diff
@@ -485,6 +497,17 @@ bool target_radio_config_set2(const struct schema_Wifi_Radio_Config *rconf,
 	if (changed->custom_options)
 		radio_config_custom_opt_set(&b, &del, rconf);
 
+	if(CSA_reuested) {
```

Comment:
I'd gather up all the test and set operations on CSA_reuested and this clause into a single function and call it here.  It will be more readable and more suitable to adding the other checks for the RRM cell-sizing step that will come next.